tutorials/stats-source-space/20_cluster_1samp_spatiotemporal.py

```diff
@@ -144,50 +144,71 @@
 X = np.abs(X)  # only magnitude
 X = X[:, :, :, 0] - X[:, :, :, 1]  # make paired contrast
 
-
 # %%
-# Compute statistic
+# Find adjacencies
 # -----------------
 #
-# To use an algorithm optimized for spatio-temporal clustering, we
-# just pass the spatial adjacency matrix (instead of spatio-temporal)
+# For cluster-based permutation testing, we must define adjacencies based
+# on which clusters can be found.
+#
+# Here, to use an algorithm optimized for spatio-temporal clustering, we
+# just pass the spatial adjacency matrix (instead of spatio-temporal).
+# But note that temporal clustering still takes place and can be
```

Comment:
nitpick, but the current wording may suggest clustering in space and time is done separately
```suggestion
# But note that clustering still takes place along the
#  temporal dimension and can be
```
sorry that the lines are off in the suggestion